scripts/unsubscribe_inactive_people.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+def unsubscribe
+  User.all.each do |user|
+    next unless user.user_contests.order(:created_at).last.created_at < Time.current - 6.months
+
+    user.user_notifications.each(&:destroy!)
+    Mailgun.send_message(
+      to: user.email,
+      text: 'Dengan email ini, kami mohon maaf atas ketidaknyamanannya.' \
```

Comment:
Please check this text

Reply:
Kami melihat bahwa Anda sudah tidak mengikuti KTOM selama 6 bulan terakhir. Oleh karena itu, kami akan stop memberikan Anda notifikasi seputar kontes di KTOM.

Jika Anda ingin kembali subscribe ke notifikasi KTOM, Anda bisa masuk ke (link). Terima kasih.